### DISCUSSION TITLE
More custom picture and gif anime matrix features

### DISCUSSION THREAD

**Discussion by ukendtkontakt** [seerge/g-helper#1942] [Ideas]
when i want to have a cool gif or something on the back of my pc i always have to download the armoury crate program again because there is just way more features in the armoury crate app and i would be cool if that gap could be a little more closed so i can completely rely on the fantastic g-helper app!!!

**Comment by seerge** [comment]
@OliverArensbach you can alreday set any GIF that you want in the app and resize / reposition it

**Reply by ukendtkontakt** [reply]
Hmmm yeah there wouldn't be any way to import the rog animations that people made into g-helper?

**Reply by ukendtkontakt** [reply]
I get it if its very hard just an idea (=

**Reply by seerge** [reply]
@OliverArensbach if you can export it as GIF from AC - then it's fine. If it's in some proprietary format, then it's simply not readable.

**Reply by Doctor0You** [reply]
Well I tried to do that, I've found where is it stored in Armory Crate: `AppData\Local\Packages\B9ECED6F.ArmouryCrate_qmba6cd70vzyy\LocalState\UserContent\737C67B6-0570-4C43-9D72-FD048D2FDAD5\matrix` It's .gif, or in .default, but you have to just rename it. And  also the gifs are positioned that, it doesn't display well in Ghelper. It's rotated and doesn't really want to match properly with anime matrix. I would love to use some gifs like that but it seems impossible to align it

**Reply by seerge** [reply]
@Doctor0You if you can upload your GIFs here , I can look at them and may be give some tips on how to fit them better.

Usually it's better to use ``Diagonal`` layout to make it close to pixel-perfect and zoom / pan accordingly

**Reply by Doctor0You** [reply]
@seerge I think on these gifs it might be visible the best, but all of them are made like that. But if it isn't on the whole display, it can be easier to have there, for example ,,Graffiti'' - ![Graffiti_GA401](https://github.com/seerge/g-helper/assets/40304283/f774bf29-ca64-4b50-9a32-334354f5064f)
![Cat_GA401](https://github.com/seerge/g-helper/assets/40304283/21072de5-538b-4bee-86ae-3e49763f9410)
![Space_GA401](https://github.com/seerge/g-helper/assets/40304283/9689bdea-2cdd-49b6-a8e8-4ded126b313e)
![Wizard_GA401](https://github.com/seerge/g-helper/assets/40304283/e7c4080a-4396-48c0-87af-3d061fae197a)

**Reply by seerge** [reply]
@Doctor0You do you use latest version ? https://github.com/seerge/g-helper/releases/tag/v0.151 they look absolutely perfect with all default automatic fitting. Just make sure to set image rotation to ``Diagonal``

**Reply by seerge** [reply]
![5EA94080-976F-4231-B8EE-93C20F1AAD2C](https://github.com/seerge/g-helper/assets/5920850/1079791d-8ac6-45dd-9322-d60008a782fe)

**Reply by Doctor0You** [reply]
@seerge Please kill me... Yea I tried the new version and it's so much better. I had to fiddle with it a bit (but it wasn't a nightmare) but it looks so nice now. Thanks <3

**Reply by ukendtkontakt** [reply]
@Doctor0You Whattt how did you even manage to find the location of the "gifs" from ac
I will definitly be trying that too

**Reply by Doctor0You** [reply]
@OliverArensbach I tried to look up all folders and files named gif, gifs, matrix, or something like that and I got into that. To be honest, there are more folders where gifs for matrix are stored, because Armory crate, but this should be files downloaded from store. The other one is `\LightingService\MatrixImage` and there are also pictures and gifs that you uploaded to a project.
